functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -199,36 +259,61 @@ Then it will export the results to Export-DbaDiagnosticQuery.
                 $QueryName = Invoke-DiagnosticQuerySelectionHelper $parsedscript
                 $first = $false
             }
+            #since some database level queries can take longer (such as fragmentation) calculate progress with database specific queries * count of databases to run against into context
+            $CountOfDatabases = ($databases).count
```

Comment:
You used this method earlier to pull this value, why not use the same method here?

`@($Databases).Count`

Reply:
Count is also capitalized because it is a property name.